robottelo/logging.py

```diff
@@ -0,0 +1,54 @@
+import os
+from logging import Formatter
+
+import yaml
+from attrdict import AttrDict
+from logzero import setup_logger
+
+from robottelo.config import get_project_root
+
+ROBOTTELO = 'robottelo'
+
+logging_yaml = AttrDict(yaml.load_all(os.path.join(get_project_root(), 'logging.yaml')))
```

Comment:
if fet_project_root returned a pathlib object..
```suggestion
logging_yaml = AttrDict(yaml.load_all(get_project_root().joinpath('logging.yaml')))
```